templates/docker-compose.yml

```diff
@@ -7,7 +7,19 @@ manager:
   image: swarm
   restart: always
   ports:
-    - 2376:2375
-  command: manage {{connection_string}}/swarm
-{% endif -%}
+    - 3376:2375
```

Comment:
Port 2376 is the TCP wrapped port (by Upstream convention)
Port 3376 is the TCP wrapped scheduler management port (by Upstream convention)